app/models/db/Users.scala

```diff
@@ -64,6 +63,26 @@ object Users extends LazyLogging {
     val query = userTable.filter(_.id === userId)
     connection.db.run(query.delete)
   }
+
+  def searchUsers(filters:Seq[TechFilter])(implicit connection: DBConnection):Future[Seq[User]] = {
+    Skills.getAllSkills.map{ skills =>
```

Comment:
I'm thinking, is it possible to make the filtering in the db directly? 
Instead of getting all skills we get only what we need